x/builder/module.go

```diff
@@ -91,6 +97,9 @@ func NewAppModule(cdc codec.Codec, keeper keeper.Keeper) AppModule {
 	}
 }
 
+// IsOnePerModuleType implements the depinject.OnePerModuleType interface.
+func (am AppModule) IsOnePerModuleType() {}
+
```

Comment:
I think we also need the following? Not sure.
```suggestion
// IsAppModule implements the appmodule.AppModule interface.
func (am AppModule) IsAppModule() {}
```